runtime/shiden/Cargo.toml

```diff
@@ -34,6 +34,10 @@ pallet-authorship = { git = "https://github.com/paritytech/substrate", branch =
 pallet-aura = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24", default-features = false }
 pallet-assets = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24", default-features = false }
 pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24", default-features = false }
+# TODO: keep features = ["unstable-interface"] ???
+pallet-contracts = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24", default-features = false } 
```

Comment:
We should not enable unstable feature as it is only for testing purpose. As specified in the pallet's Cargo
https://github.com/paritytech/substrate/blob/16bae92b3f257f9c15525ae3d0a23d036bd59715/frame/contracts/Cargo.toml#L90

Reply:
Oh thanks, I haven't seen this :)
I only assumed that xD